### ISSUE TITLE
Hyperlinked numeric references probably shouldn't get converted to e.g. ![[67]]

### ISSUE THREAD

**Issue by NathanSkene** [akosbalasko/yarle#50]
Sorry to spam you with issues today but excited to try and get the library exported and having some issues

I've got lots of notes that have scientific article text pasted in. E.g. the below image

![image](https://user-images.githubusercontent.com/33265555/93503221-b08bd080-f90f-11ea-8681-c66e2a3793eb.png)

When run through your script this gets converted as follows:

```
Different roles in learning?

An obvious implication of the co-existence of multiple forms of LTP is that their roles in learning and memory processing might be dissociable. Tantalizing clues that this might, indeed, be the case are provided by a brief review of recent hippocampus-dependent behavior studies. With respect to LTP3, impaired function of L-type VDCCs seems to selectively impair long-term memory beyond a few days, without an effect on acquisition or working memory ![[67]], ![[78]] and ![[79]]. Similarly, interference with CRE-mediated transcription, the cAMP and PKA pathway and the ERK–MAPK pathway primarily affects long-term memory ![[2]] and ![[80]]. Correlating to LTP2, inhibition of Group I mGlu receptors seems to selectively impair reference memory during the period (24–48 h) between acquisition trials ![[81]] and ![[82]]. In relation to LTP1-specific mechanisms, genetic deletion of Ry receptors produces an intriguing behavior phenotype in which acquisition and long-term memory of a spatial learning task are intact, but the ability to subsequently learn a new arrangement of the same task is impaired ![[83]] and ![[84]]. The application of the LTP1, LTP2 and LTP3 framework could greatly assist in the design and interpretation of future investigations into the role of LTP in different aspects of learning and memory.
```

Obsidian then does something really wierd and converts this to the following:

```
Different roles in learning?

An obvious implication of the co-existence of multiple forms of LTP is that their roles in learning and memory processing might be dissociable. Tantalizing clues that this might, indeed, be the case are provided by a brief review of recent hippocampus-dependent behavior studies. With respect to LTP3, impaired function of L-type VDCCs seems to selectively impair long-term memory beyond a few days, without an effect on acquisition or working memory ![[unknown_filename.67.png]], ![[WhiskyHighland_Basic_Valuation_Port_Ellen_1978.pdf]] and ![[201907241020 heritability of alzheimers disease 79%]]. Similarly, interference with CRE-mediated transcription, the cAMP and PKA pathway and the ERK–MAPK pathway primarily affects long-term memory ![[201107121226 golgi]] and ![[201108180400 pw]]. Correlating to LTP2, inhibition of Group I mGlu receptors seems to selectively impair reference memory during the period (24–48 h) between acquisition trials ![[201107130819 jim eberwine, march 2011, neuron pnas, dec 2010]] and ![[201208261127 paid]]. In relation to LTP1-specific mechanisms, genetic deletion of Ry receptors produces an intriguing behavior phenotype in which acquisition and long-term memory of a spatial learning task are intact, but the ability to subsequently learn a new arrangement of the same task is impaired ![[201202010839 jobs to apply for_]] and ![[201201040845 earliest ferry from ryde on offpeak]]. The application of the LTP1, LTP2 and LTP3 framework could greatly assist in the design and interpretation of future investigations into the role of LTP in different aspects of learning and memory.
```

I don't really understand why Obsidian decides to swap e.g. ![[67]] for ![[WhiskyHighland_Basic_Valuation_Port_Ellen_1978.pdf]]  but a good start might be to just not convert numeric values (even if they are hyperlinks) into [[...]] links. It seems to have might a right tangled mess of the knowledge graph as many notes have these!

Thanks again

**Comment by akosbalasko** [comment]
Hi @NathanSkene ! No worries, I'm glad you write these reports because in this way we can improve to code to be better and better. I would investigate this issue a bit more if you don't mind. Could you please send me the example note that failed to be transformed? 
Thanks a lot!

**Comment by akosbalasko** [comment]
Hi @NathanSkene !
I released the fix (2.9.1), please give it a try.
